docs/AppBar.md

```diff
@@ -0,0 +1,40 @@
+# App Bar
+
+The `<pxb-app-bar>` component is a wrapper around the `<mat-toolbar>` that can be resized as the page is scrolled. It supports three modes:
+
+## Usage
+
+```tsx
+// Default: AppBar will resize between collapsedHeight and expandedHeight as the window is scrolled
+<pxb-app-bar mode='dynamic'></pxb-app-bar>
+
+// App Bar will stay fixed at the collapsedHeight size
+// Default: AppBar will resize between collapsedHeight and expandedHeight as the window is scrolled
+<pxb-app-bar mode='collapsed'></pxb-app-bar>
+
+// App Bar will stay fixed at the expandedHeight size
+// Default: AppBar will resize between collapsedHeight and expandedHeight as the window is scrolled
+<pxb-app-bar mode='expanded'></pxb-app-bar>
+```
+
+## API
+
+<div style="overflow: auto;">
+
+| Prop Name              | Description                                  | Type                                         | Required | Default       |
```

Comment:
Table format does not match what we use for our angular components

Reply:
My bad! That was left over from the react docs.   It's updated now.